app/src/main/java/org/onionshare/android/tor/TorManager.kt

```diff
@@ -64,118 +55,80 @@ class TorManager @Inject constructor(
      * Starts [ShareService] and creates a new onion service.
      * Suspends until the address of the onion service is available.
      */
+    @OptIn(DelicateCoroutinesApi::class)
+    @Throws(IOException::class, IllegalArgumentException::class, InterruptedException::class)
     suspend fun start() = withContext(Dispatchers.IO) {
-        if (state.value !is TorState.Stopped) stop()
-
         LOG.info("Starting...")
         val now = System.currentTimeMillis()
-        _state.value = TorState.Starting(progress = 0, startTime = now, lastProgressTime = now)
-
+        _state.value = TorState.Starting(progress = 0, lastProgressTime = now)
         Intent(app, ShareService::class.java).also { intent ->
             startForegroundService(app, intent)
         }
 
         tor.start()
+        changeStartingState(5)
+        if (settingsManager.automaticBridges.value) {
+            // we try without bridges first
+            tor.disableBridges()
```

Comment:
TorWrapper starts with a fresh config each time and doesn't use bridges by default, so I think this should be redundant.